projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -356,8 +357,7 @@ public static function parse( $attributes, $content ) {
 				$form_classes .= ' wp-block-jetpack-contact-form';
 			}
 
-			$r .= "<form action='" . esc_url( $url ) . "' method='post' class='" . esc_attr( $form_classes ) . "'>\n";
-			$r .= self::get_script_for_form();
```

Comment:
`get_script_for_form` used to disable the submit button once it was clicked. I suppose this was to prevent users from submitting the form multiple times, but it now prevents users from correcting the form.